README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
The jars that are built are for the ElasticSearch and InfluxDB Java API's needed for the MetricsSink operators to communicate and add data to the databases. There's quite a few jars (~30 MB, ~50 jars) which is why I thought excluding them would be for the best.

If I include them, it would definitely make it easier on the users, though.